src/components/Header/index.js

```diff
@@ -1,97 +1,108 @@
-import React, { useState } from 'react'
+import React, { Fragment, useState, useEffect } from 'react'
 
 import styled from 'styled-components'
 import Flex from 'styled-flex-component'
 import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
 
+import navLinks from './navLinks'
 import { Row, Col, Grid } from '../grid'
 import Hamburger from './Hamburger'
 import Overlay from './Overlay'
 import TopNav from './TopNav'
 import SideNav from './SideNav'
 
-const FixedHeightFlex = styled(Flex)`
-  height: 84px;
+const StyledPadding = styled(Padding)`
+  display: flex;
+  height: ${remcalc(120)};
+  align-items: center;
 `
 
-const navLinks = [
-  {
-    label: 'Services',
-    dropdownItems: [
-      {
-        label: 'Engineering',
-        to: '/engineering/'
-      },
-      {
-        label: 'Design',
-        to: '/design/'
-      },
-      {
-        label: 'Training',
-        to: '/training/'
-      }
-    ]
-  },
-  {
-    label: 'Our work',
-    to: '/our-work/'
-  },
-  {
-    label: 'Blog',
-    href: 'https://medium.com/yld-engineering-blog/'
-  },
-  {
-    label: 'About',
-    dropdownItems: [
-      {
-        label: 'Our team',
-        to: '/about-us/'
-      },
-      {
-        label: 'Contact',
-        to: '/contact/'
-      }
-    ]
-  },
-  {
-    label: 'Join us',
-    to: '/join-us/'
-  }
-]
+const StyledGrid = styled(Grid)`
+  position: fixed;
+  height: ${remcalc(84)};
+  background: ${props =>
+    props.theme.colors[props.isSpecialityPage ? 'blueBg' : 'white']};
+  width: 100%;
+  max-width: unset;
+  z-index: 100;
+
+  ${breakpoint('smallTablet')`
+    max-width: none;
+    margin: 0 auto;
+  `}
+
+  ${breakpoint('desktop')`
+    right: 50%;
+    margin: 0 -50% 0 0;
+    display: flex;
+    justify-content: center;
+  `}
+`
+
+const StyledShadowGradient = styled.div`
```

Comment:
I was using box-shadow at first as you know but could not get the transparency effect with it - that's why I had to go to a linear gradient div instead; and achieve the below:
<img width="422" alt="Screen Shot 2019-05-02 at 11 18 22" src="https://user-images.githubusercontent.com/10553003/57069299-1e5c3b80-6ccc-11e9-9fec-8ae68e354e79.png">

Reply:
This could probably be achieved via `rgba` rather than a hexcode, but still using box-shadow CSS? https://stackoverflow.com/questions/5458716/is-it-possible-to-set-transparency-in-css3-box-shadow

Reply:
Yep. Just updated back to box shadow with `0 9px 9px -9px rgba(0, 0, 0, 0.175)` which is close to what we have now. Will ask feedback from designers.